src/main/webapp/app/shared/alert/alert.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="alerts mt-1" role="alert">
+    <div *ngFor="let alert of alerts" [ngClass]="setClasses(alert)">
+        <ngb-alert *ngIf="alert && alert.type && alert.message && alert.close" [type]="alert.type" (close)="close(alert)">
+            <pre [innerHTML]="getAlertMessage(alert)"></pre>
```

Comment:
I added this to the open[ points in the confluence page](https://confluence.ase.in.tum.de/display/ArTEMiS/Code+Review), this might be a follow-up task for the whole code base, as this is not the only place where we call methods inside a template.